pkg/controller/infrastructure/infraflow/ensure.go

```diff
@@ -612,3 +632,34 @@ func (fctx *FlowContext) ensureKubernetesRoutesDeleted(ctx context.Context) erro
 
 	return nil
 }
+
+func (fctx *FlowContext) ensureAliasIpRanges(ctx context.Context) error {
+	log := shared.LogFromContext(ctx)
+
+	for i, route := range fctx.state.Routes {
+		// if machine is already gone procceed with next route
+		if err := fctx.computeClient.InsertAliasIPRoute(ctx, route, DefaultSecondarySubnetName); err != nil && !strings.Contains(err.Error(), "notFound") {
```

Comment:
Can we somehow handle the case where the machine already has an aliasIP much less the same in this or the client function ? I suspect this will come up one way or the other

Reply:
Adding the same aliasIP again won't return in an error. It will just stay as it is.